c10/core/WrapDimMinimal.h

```diff
@@ -12,11 +12,11 @@ namespace detail {
 // This template can only be specialized at int64_t and c10::SymInt;
 // you'll get linker errors otherwise
 template <typename T>
-C10_API T maybe_wrap_dim_slow(T dim, T dim_post_expr, bool wrap_scalar);
+C10_API T maybe_wrap_dim_slow(T dim, const T& dim_post_expr, bool wrap_scalar);
```

Comment:
The other one makes sense since it a private function, this one I'd be concerned about overloading / SFINAE errors.